tests/e2e/cucumber/environment/index.ts

```diff
@@ -93,14 +95,18 @@ After(async function (this: World, { result }: ITestCaseHookParameter) {
 
   await this.attach(`Status: ${result?.status}. Duration:${result.duration?.seconds}s`)
 
-  if (result.status !== Status.PASSED) {
+  if (result.status !== Status.PASSED && retryCounter < config.retry) {
     if (result.willBeRetried) {
       config.reportHar = true
       config.reportTracing = true
+      retryCounter += 1
```

Comment:
Why not set it to 1 hardcoded?

Reply:
then it won't work when RETRY=2 and more